sirepo/simulation_db.py

```diff
@@ -1151,11 +1264,18 @@ def _validate_name_uniquify(data, starts_with):
     data.models.simulation.name = n2
 
 
-
-# Ensure the value of a numeric field falls within the supplied limits (if any)
-# Note that currently the values in enum arrays at the indices below are sometimes
-# used for other purposes, so we return for non-numeric values rather than fail
 def _validate_number(val, sch_field_info):
+    """Ensure the value of a numeric field falls within the supplied limits (if any)
+
+    Note that currently the values in enum arrays at the indices below are sometimes
```

Comment:
This comment should be at the place in the code, since it is not relevant to the description of the function as whole.